### DISCUSSION TITLE
Is higher image resolution or batch size better?

### DISCUSSION THREAD

**Discussion by anodyne99** [ultralytics/yolov5#7300] [Q&A] [answered]
I am currently working on a research project for an AI course and have been tuning the model training to try to get the best results.

However, I'm limited in my range for training parameters due to hardware. My training dataset is roughly 11k images running the gamut of stock and surveillance photos. In some of them the desired objects to detect in the image are quite small so I increased the resolution during training to 1024, however I run out of memory if I set the batch size to anything larger than 32.

From what I've read in the discussions and documentation, the recommended batch size is at least 64. Should I lower the resolution until I can get to the 64 batch size or should I keep it at 32 to have the larger image resolution?

Essentially what I'm asking is which is more important in general for best results? Batch size, image resolution, or does it depend on the situation?

**Comment by glenn-jocher** [comment, marked as answer]
@anodyne99 see batch size study https://github.com/ultralytics/yolov5/discussions/2452

**Reply by anodyne99** [reply]
Thank you sir, I didn't see this.

**Comment by philipakomolafe** [comment]
Good read..
